### ISSUE TITLE
Output for regression model

### ISSUE THREAD

**Issue by podturkinalex** [gradio-app/gradio#2182]
### Describe the bug

Hi there,
The program gives me an error when I try to upload an image with a face and make a predictions of age from the picture vith TensorFlow model: "ValueError: Component must provided as a `str` or `dict` or `Component` but is <bound method Kernel .raw_input of <google.colab._kernel.Kernel object at 0x7fd03cf2fe90>>"

I have numpy.float32 as an output. I also tried converting the output to string format as given in the error, but that didn't help either.

Please, let me know, what the output should be for my model instead. 

Below, you can find a code.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

` ## model prediction
def predict_image(input):
       array = np.input(image.resize((224, 224)))
       array = array/255

       array = np.expand_dims(array, axis=0)
       prediction = model.predict(array)
       prediction = predicted[0][0]
      return prediction

##gradio interface 

import gradio as gr
inputs=gr.Image(shape=(224, 224))
output = gr.outputs.Textbox()
gr.Interface(fn=predict_image,
              inputs=input,
              outputs=output).launch();
`

### Logs

```shell
----------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-39-c5fb949f3a3f> in <module>
      4 gr.Interface(fn=predict_image,
      5               inputs=input,
----> 6               outputs=output).launch(debug=True);

2 frames
/usr/local/lib/python3.7/dist-packages/gradio/interface.py in __init__(self, fn, inputs, outputs, examples, cache_examples, examples_per_page, live, interpretation, num_shap, title, description, article, thumbnail, theme, css, allow_flagging, flagging_options, flagging_dir, flagging_callback, analytics_enabled, _api_mode, **kwargs)
    247 
    248         self.input_components = [
--> 249             get_component_instance(i, render=False) for i in inputs
    250         ]
    251         self.output_components = [

/usr/local/lib/python3.7/dist-packages/gradio/interface.py in <listcomp>(.0)
    247 
    248         self.input_components = [
--> 249             get_component_instance(i, render=False) for i in inputs
    250         ]
    251         self.output_components = [

/usr/local/lib/python3.7/dist-packages/gradio/components.py in get_component_instance(comp, render)
   3983     else:
   3984         raise ValueError(
-> 3985             f"Component must provided as a `str` or `dict` or `Component` but is {comp}"
   3986         )
   3987 

ValueError: Component must provided as a `str` or `dict` or `Component` but is <bound method Kernel.raw_input of <google.colab._kernel.Kernel object at 0x7fd03cf2fe90>>
```

### System Info

```shell
gradio.__version__
3.2

Chrome
Version 104.0.5112.102 (Official Build) (64-bit)
```

### Severity

annoying

**Comment by abidlabs** [comment]
Hi @podturkinalex, it's a little hard to parse through the code. If you could kindly format the code and provide a minimal example that fully allows us to reproduce the problem, it will make it easier for us to help. 

But in the meantime, I did notice one problem, which might explain the issue you are seeing. You define `inputs=gr.Image(shape=(224, 224))` yet you use the variable `input` instead of `inputs` here: 

```py
gr.Interface(fn=predict_image,
inputs=input,
outputs=output).launch();
```

This should fix the error that you are seeing here.

**Comment by podturkinalex** [comment]
@abidlabs, thank you so much for your rapid answer and for fixing my issue!
